include/socgraph.php

```diff
@@ -11,6 +11,9 @@
 use Friendica\Core\Config;
 use Friendica\Network\Probe;
 
+use DomXPath;
```

Comment:
Actually this use statement is useless since the file isn't in a namespace. This error is thrown:
` PHP Warning:  The use statement with non-compound name 'DomXPath' has no effect in include/socgraph.php on line 14`